### PR TITLE
feature/ttnn-visualizer: combine work streams

### PR DESCRIPTION
**Pull request by dcblundell**

### CODE REVIEW

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import './styles.scss';
+
+const GOLDEN_GLOBAL = [1, 0.9999, 0.9888, 1, 1, 0.8888];
```
Comment (aidemsined) [inline ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx:3]: i understand this is our mock data but a component will/should not be aware of data that doesn't realte to it. let move mock data out of this one and pass only hat is needed

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                style={{
+                    backgroundColor: calculateOpPerformanceColor(source[index]),
+                }}
+            />
+            <span>t{source[index].toFixed(4)}</span>
+        </>
+    );
+}
+
+const calculateOpPerformanceColor = (value: number): string => {
```
Comment (aidemsined) [inline ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx:29]: this probably belongs in utils/ of sort as we would likely be using that color functionality other places

File: ttnn-visualizer/src/scss/_colorSchemeLight.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+// Unusable as it is
+@media (prefers-color-scheme: light) {
```
Comment (aidemsined) [inline ttnn-visualizer/src/scss/_colorSchemeLight.scss:5]: excellent but the rest of the app should respond to color scheme as well. we cant only control bg color. i suggest stickign to theme dark for now and potentially resurection this in the future as a single improvement feature

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/styles.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.performance-square {
```
Comment (aidemsined) [inline ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/styles.scss:1]: if the scss file is not scoped to only the component we shoudl consider namespacing to not pollute

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx (hunk 2 above)
Comment (dcblundell) [reply ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx:29]: Moved to a functions folder.

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/styles.scss (hunk 4 above)
Comment (dcblundell) [reply ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/styles.scss:1]: Wrote a more sensible class name.

File: ttnn-visualizer/src/scss/_colorSchemeLight.scss (hunk 3 above)
Comment (dcblundell) [reply ttnn-visualizer/src/scss/_colorSchemeLight.scss:5]: Commented this all out but added a todo.

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx (hunk 1 above)
Comment (dcblundell) [reply ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/index.tsx]: Moved into a mock object.

File: ttnn-visualizer/src/components/GoldenTensorComparisonIndicator.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import calculateOpPerformanceColor from '../functions/calculateOpPerformanceColor';
+// TODO: look at making this import path nicer (alias throw a linting error in .tsx files)
+import '../scss/components/GoldenTensorComparisonIndicator.scss';
+
+const GOLDEN_GLOBAL = [1, 0.9999, 0.9888, 1, 1, 0.8888];
```
Comment (aidemsined) [inline ttnn-visualizer/src/components/GoldenTensorComparisonIndicator.tsx:5]: we ~probably~ definitely dont need those

Comment (dcblundell) [reply ttnn-visualizer/src/components/GoldenTensorComparisonIndicator.tsx]: 🔪

**Review by aidemsined — APPROVED** [review]

**Pull request merged**